### PR TITLE
feat(domain.general.informations): add shortcut to manage alldom

### PR DESCRIPTION
**Pull request by TedKac**
## feat(domain.general.informations): add shortcut to manage alldom

MBP-425

### CODE REVIEW

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -17,6 +17,12 @@
                             </li>
                         </ul>
                     </oui-tile-description>
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item
+                            data-text="{{ ::'status_action_renew_management' | translate }}"
```
Comment (antleblanc) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: `text` attribut is now deprecated. I roughly encourage you to use the transclude slot like this:

```html
<oui-action-menu
    data-compact
    data-align="end">
    <oui-action-menu-item data-href="{{ ::$ctrl.actions.manageAlldom.href }}">
        <span data-ng-bind="{{ ::'status_action_renew_management' | translate }}"></span>
    </oui-action-menu-item>
</oui-action-menu>
```

File: client/app/domain/general-informations/domain-general-informations.controller.js [hunk 2]
```diff
@@ -150,6 +150,13 @@ angular.module('App').controller(
           }&selectedType=DOMAIN`,
           isAvailable: () => true,
         },
+        manageAlldom: {
+          text: this.$translate.instant('common_manage'),
```
Comment (antleblanc) [inline client/app/domain/general-informations/domain-general-informations.controller.js]: It sounds a bit strange to have the same text as [`manageAutorenew.text` L147](https://github.com/ovh-ux/ovh-manager-web/pull/990/files#diff-173bd65a8e102b59a4e7f07cda4a8979R147).

Furthermore you are only using the `href` attribut so you can get rid of both `text` and `isAvailable` properties.

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (marie-j) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: `text` attribute is deprecated, prefer using a nested span 
```suggestion
  <oui-action-menu-item>
    <span data-translate="status_action_renew_management"></span>
  </oui-actio-menu-item>
```

Comment (TedKac) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: Fixed by using span +  data-translate

File: client/app/domain/general-informations/domain-general-informations.controller.js (hunk 2 above)
Comment (TedKac) [reply client/app/domain/general-informations/domain-general-informations.controller.js]: Fixed

**Pull request merged**